[2024-11-02T23:24:39.189+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-02T23:24:39.324+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection.process_transaction scheduled__2024-11-02T23:22:00+00:00 [queued]>
[2024-11-02T23:24:39.406+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection.process_transaction scheduled__2024-11-02T23:22:00+00:00 [queued]>
[2024-11-02T23:24:39.408+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-11-02T23:24:39.498+0000] {taskinstance.py:2888} INFO - Executing <Task(BranchPythonOperator): process_transaction> on 2024-11-02 23:22:00+00:00
[2024-11-02T23:24:39.551+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=855) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-02T23:24:39.554+0000] {standard_task_runner.py:72} INFO - Started process 865 to run task
[2024-11-02T23:24:39.565+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fraud_detection', 'process_transaction', 'scheduled__2024-11-02T23:22:00+00:00', '--job-id', '60', '--raw', '--subdir', 'DAGS_FOLDER/fraud_detection_dag.py', '--cfg-path', '/tmp/tmpxg10bufh']
[2024-11-02T23:24:39.584+0000] {standard_task_runner.py:105} INFO - Job 60: Subtask process_transaction
[2024-11-02T23:24:40.077+0000] {task_command.py:467} INFO - Running <TaskInstance: fraud_detection.process_transaction scheduled__2024-11-02T23:22:00+00:00 [running]> on host 713744f3b6c0
[2024-11-02T23:24:40.897+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='fraud_team' AIRFLOW_CTX_DAG_ID='fraud_detection' AIRFLOW_CTX_TASK_ID='process_transaction' AIRFLOW_CTX_EXECUTION_DATE='2024-11-02T23:22:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-02T23:22:00+00:00'
[2024-11-02T23:24:40.898+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-02T23:24:40.973+0000] {baseoperator.py:405} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-11-02T23:24:47.427+0000] {fraud_detection_dag.py:115} ERROR - Erreur lors du traitement: columns are missing: {'unix_time'}
[2024-11-02T23:24:47.430+0000] {python.py:240} INFO - Done. Returned value was: skip_processing
[2024-11-02T23:24:47.431+0000] {branch.py:38} INFO - Branch into skip_processing
[2024-11-02T23:24:47.432+0000] {skipmixin.py:230} INFO - Following branch ('skip_processing',)
[2024-11-02T23:24:47.547+0000] {skipmixin.py:278} INFO - Skipping tasks [('notify_normal', -1), ('notify_fraud', -1)]
[2024-11-02T23:24:47.862+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-02T23:24:47.863+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=fraud_detection, task_id=process_transaction, run_id=scheduled__2024-11-02T23:22:00+00:00, execution_date=20241102T232200, start_date=20241102T232439, end_date=20241102T232447
[2024-11-02T23:24:48.265+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-02T23:24:48.863+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-02T23:24:48.946+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
